solvation_analysis/solvation.py

```diff
@@ -114,7 +116,14 @@ def get_closest_n_mol(u, central_species, n_mol=5, radius=3):
     partial_shell = u.select_atoms(f"point {str_coords} {radius}")
```

Comment:
This calls `capped_distance` under the hood, so we should use it directly. It can also return the distances in one shot rather than recalculating them below.

Reply:
Thanks! this is much better.